raiden-dapp/Gemfile.lock

```diff
@@ -7,7 +7,7 @@ GEM
     artifactory (3.0.15)
     atomos (0.1.3)
     aws-eventstream (1.1.1)
-    aws-partitions (1.483.0)
```

Comment:
@weilbith Should we update this? I remember vaguely that this was important to have exactly the same version for something.

Reply:
Aeh... You could test if it still works. ^^
What are you referring to with the same version? 🤔

Reply:
You don't mix this up with the resource repos using Packer and Terraform? 
Do be honest I don't know why there is a dependency for AWS. :shrug:

Reply:
Mh, I'm probably wrong.

In my head you mentioned at some point that fastlane is really picky about versions... So I assumed we might have to do some manual tasks to update the gemfile.

Reply:
Thaaaat could be. ^^
But Yeah, after all we have no tests for this (and there basically can't) so we kinda have no other option then trying it. Sorry. Maybe push the PR back and do it later. Shouldn't matter now.